companion/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT( companion )
```

Comment:
Isn't it possible to have a different name for the package / directories, without changing the project name? It seems not good to me to do so!